frontend/src/pages/Admin/Flags.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { Link } from 'react-router-dom';
+import Container from '../../components/Container';
+import { getFeatures } from '../../fetchers/Admin';
+import './Flags.css';
+
+export default function Flags() {
+  const [features, setFeatures] = useState();
+  const [error, setError] = useState();
+
+  useEffect(() => {
+    async function fetchFeatures() {
+      try {
+        const featuresFromApi = await getFeatures();
+        setFeatures(featuresFromApi);
+      } catch (e) {
+        setError('Unable to fetch features');
+      }
+    }
+
+    if (!features) {
+      fetchFeatures();
+    }
+  }, [features]);
```

Comment:
If you give this `useEffect` an empty dependency array you can drop the check on features existing